hack/build-cross.sh

```diff
@@ -12,6 +12,11 @@ if [[ -n "${OS_ONLY_BUILD_PLATFORMS-}" ]]; then
       filtered+=("${platform}")
     fi
   done
+  if [[ ${OS_ONLY_BUILD_PLATFORMS} == "linux/ppc64le" ]]; then
```

Comment:
We should handle these two cases by making sure `"${OS_CROSS_COMPILE_PLATFORMS}"` contains these architectures when we can build for them, as we are doing in [`common.sh`](https://github.com/openshift/origin/blob/master/hack/common.sh#L50-L54)